cmd/ketch/output/output.go

```diff
@@ -0,0 +1,37 @@
+package output
+
+import (
+	"io"
+
+	"github.com/spf13/pflag"
+)
+
+type Writer interface {
```

Comment:
I don't think we need to export this interface, it's used only by the `Write` function.
it can be 
```golang
	type writer interface {
		Write() error
	}
	var w writer

```

Reply:
Change made.